apis/swagger.yml

```diff
@@ -947,16 +947,449 @@ definitions:
   SystemInfo:
     type: "object"
     properties:
-      Architecture:
+      ID:
+        description: |
+          Unique identifier of the daemon.
+
+          <p><br /></p>
+
+          > **Note**: The format of the ID itself is not part of the API, and
+          > should not be considered stable.
         type: "string"
+        example: "7TRN:IPZB:QYBB:VPBQ:UMPP:KARE:6ZNR:XE6T:7EWV:PKF4:ZOJD:TPYS"
       Containers:
+        description: "Total number of containers on the host."
         type: "integer"
+        example: 14
       ContainersRunning:
+        description: |
+          Number of containers with status `"running"`.
         type: "integer"
+        example: 3
+      ContainersPaused:
+        description: |
+          Number of containers with status `"paused"`.
+        type: "integer"
+        example: 1
       ContainersStopped:
+        description: |
+          Number of containers with status `"stopped"`.
         type: "integer"
-      ContainersPaused:
+        example: 10
+      Images:
+        description: |
+          Total number of images on the host.
+
+          Both _tagged_ and _untagged_ (dangling) images are counted.
         type: "integer"
+        example: 508
+      Driver:
+        description: "Name of the storage driver in use."
+        type: "string"
+        example: "overlay2"
+      DriverStatus:
+        description: |
+          Information specific to the storage driver, provided as
+          "label" / "value" pairs.
+
+          This information is provided by the storage driver, and formatted
+          in a way consistent with the output of `pouch info` on the command
+          line.
+
+          <p><br /></p>
+
+          > **Note**: The information returned in this field, including the
+          > formatting of values and labels, should not be considered stable,
+          > and may change without notice.
+        type: "array"
+        items:
+          type: "array"
+          items:
+            type: "string"
+        example:
+          - ["Backing Filesystem", "extfs"]
+          - ["Supports d_type", "true"]
+          - ["Native Overlay Diff", "true"]
+      PouchRootDir:
+        description: |
+          Root directory of persistent Pouch state.
+
+          Defaults to `/var/lib/pouch` on Linux, and `C:\ProgramData\pouch`
```

Comment:
we may just keep `Defaults to '/var/lib/pouch'  on Linux`, because we just support Linux right now, :) WDYT ?

Reply:
Right :)